data/hails.txt

```diff
@@ -329,13 +337,19 @@ phrase "friendly civilian"
 		"giving their money to"
 		"paying to listen to"
 		"so in love with"
+		"streaming"
+		"on about"
```

Comment:
Doesn't work: "...they said they're on about that group..."

Reply:
what doesn't work with that? someone said that someone else keeps going on about some group they dont care about. Just because you don't speak a certain way, doesn't mean no one does. Stop being so critical dude.

Reply:
If it's supposed to be "going on about," then you need to add the word "going."

Reply:
("On about" is a valid expression in itself, such as in, "What are you on about?" but in this situation, only "going on about" works.)